Dockerfile.Windows

```diff
@@ -24,6 +24,8 @@ RUN cd /code/ && GOARCH=$(echo $TARGETPLATFORM | cut -f2 -d '/') GCE_PD_CSI_STAG
 
 FROM ${BASE_IMAGE}
```

Comment:
A followup item is to use the new base image that sig-windows created in https://static.sched.com/hosted_files/kccncna2022/c6/HPC-talk.pdf